Classes/Command/RunTestsCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TYPO3 CMS-based extension "b13/make" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+namespace B13\Make\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+use TYPO3\CMS\Core\Package\PackageInterface;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Command for creating docker based test environment for a TYPO3 extension
+ */
+class RunTestsCommand extends AbstractCommand
```

Comment:
Have you tried to build this using the SimpleComponentCommand "API"? I'm asking because this now introduces dependencies to Symfony's filesystem and finder, which are as of today, not required by this extension.

Reply:
Restructured the code to avoid filesystem and finder and trying to use SimpleComponentCommand.